src/config/mod.rs

```diff
@@ -78,6 +78,9 @@ pub struct Config {
     // Enable/disable notification replace functionality.  I don't like how some apps do it.
     #[serde(default = "maths_utility::val_true")]
     pub replacing_enabled: bool,
+    // Whether or not to refresh the timeout of a notification on an update
+    #[serde(default = "maths_utility::val_false")]
```

Comment:
For bools that should be false by default, we can just use of `bool`s [`Default` impl](https://doc.rust-lang.org/std/primitive.bool.html#impl-Default).

```suggestion
    #[serde(default)]
```